tests/integrations/deploy/test_api_backend.py

```diff
@@ -50,9 +52,8 @@ def test_integration_deploy_existing_function(scaleway_project: str):  # noqa
         app_path=constants.APP_FIXTURES_PATH / "app_updated.py",
     )
 
-    import time
-
-    time.sleep(60)
+    # TODO?: delete this
+    time.sleep(30)
```

Comment:
With sleeps like this it's cleaner to add a loop that retries up to some limit. Perhaps we can add a utility function to trigger a function, then sleep for 5 seconds and retry, up to 10 retries?